diptest/phase_test.go

```diff
@@ -92,6 +145,353 @@ func TestStartGame(t *testing.T) {
+					"gal",
+					"bud",
+				},
+				{
+					"bul",
+					"Move",
+					"bla",
+				},
+				{
+					"sev",
+					"Move",
+					"arm",
+				},
+			},
+		},
+	},
+	{},
+	{
+		{
+			nat: "Russia",
+			ord: [][]string{
+				{
+					"rum",
+					"Move",
+					"ser",
+				},
+				{
+					"bla",
+					"Support",
+					"arm",
+					"ank",
+				},
+				{
+					"arm",
+					"Move",
+					"ank",
+				},
+			},
+		},
+	},
+	{},
+	{
+		{
+			nat: "Russia",
+			ord: [][]string{
+				{
+					"stp/nc",
+					"Build",
+					"Fleet",
+				},
+				{
+					"war",
+					"Build",
+					"Army",
+				},
+				{
+					"sev",
+					"Build",
+					"Army",
+				},
+				{
+					"mos",
+					"Build",
+					"Army",
+				},
+			},
+		},
+	},
+	{
+		{
+			nat: "Russia",
+			ord: [][]string{
+				{
+					"stp",
+					"Move",
+					"nwy",
+				},
+				{
+					"nwy",
+					"Move",
+					"nth",
+				},
+				{
+					"den",
+					"Move",
+					"bal",
+				},
+				{
+					"war",
+					"Move",
+					"gal",
+				},
+				{
+					"mos",
+					"Move",
+					"war",
+				},
+				{
+					"bud",
+					"Support",
+					"ser",
+					"tri",
+				},
+				{
+					"ser",
+					"Move",
+					"tri",
+				},
+				{
+					"sev",
+					"Move",
+					"rum",
+				},
+				{
+					"ank",
+					"Support",
+					"bla",
+					"con",
+				},
+				{
+					"bla",
+					"Move",
+					"con",
+				},
+			},
+		},
+	},
+	{},
+	{
+		{
+			nat: "Russia",
+			ord: [][]string{
+				{
+					"nwy",
+					"Move",
+					"nrg",
+				},
+				{
+					"bal",
+					"Support",
+					"pru",
+					"ber",
+				},
+				{
+					"pru",
+					"Move",
+					"ber",
+				},
+				{
+					"war",
+					"Move",
+					"sil",
+				},
+				{
+					"gal",
+					"Move",
+					"vie",
+				},
+				{
+					"bud",
+					"Support",
+					"gal",
+					"vie",
+				},
+				{
+					"rum",
+					"Move",
+					"ser",
+				},
+				{
+					"con",
+					"Support",
+					"ank",
+					"smy",
+				},
+				{
+					"ank",
+					"Move",
+					"smy",
+				},
+			},
+		},
+	},
+	{},
+	{},
+	{
+		{
+			nat: "Russia",
+			ord: [][]string{
+				{
+					"nrg",
+					"Move",
+					"edi",
+				},
+				{
+					"nth",
+					"Support",
+					"nrg",
+					"edi",
+				},
+				{
+					"bal",
+					"Move",
+					"kie",
+				},
+				{
+					"ber",
+					"Support",
+					"bal",
+					"kie",
+				},
+				{
+					"vie",
+					"Move",
+					"tyr",
+				},
+				{
+					"bud",
+					"Move",
+					"vie",
+				},
+				{
+					"con",
+					"Move",
+					"aeg",
+				},
+			},
+		},
+	},
+	{},
+}
+
+func TestSoloEnding(t *testing.T) {
+	withStartedGameOptsAndOrders(nil, russianSoloOrders, func() {
+		orderSet{
+			{
+				nat:  "England",
+				dias: true,
+			},
+		}.execute(len(russianSoloOrders) + 1)
+		game := startedGameEnvs[0].GetRoute("Game.Load").RouteParams("id", startedGameID).Success()
+		game.AssertBoolEq(true, "Properties", "Finished")
+		game.Follow("game-result", "Links").Success().AssertNil("Properties", "DIASMembers").AssertEq("Russia", "Properties", "SoloWinnerMember")
```

Comment:
Is there a list of "Losers" too?  If so then is it worth checking England is present?

Reply:
No loser list, it's just assumed that if someone wins, every one else loses :D